extensions/smallrye-jwt/runtime/src/main/java/io/quarkus/smallrye/jwt/runtime/auth/JWTAuthMechanism.java

```diff
@@ -42,8 +44,13 @@
             IdentityProviderManager identityProviderManager) {
         String jwtToken = new VertxBearerTokenExtractor(authContextInfo, context).getBearerToken();
         if (jwtToken != null) {
+            TokenAuthenticationRequest tokenAuthenticationRequest = new TokenAuthenticationRequest(
```

Comment:
I'm sorry, this is not a bad workaround but I do believe we need to sort out an authentication mechanism alignment instead, I'll open an issue right away and CC you